neonvm/controllers/virtualmachine_controller.go

```diff
@@ -430,6 +430,10 @@ func (r *VirtualMachineReconciler) doReconcile(ctx context.Context, virtualmachi
 				msg = fmt.Sprintf("%s, SSH Secret %s", msg, sshSecret.Name)
 			}
 			r.Recorder.Event(virtualmachine, "Normal", "Created", msg)
+			if !virtualmachine.HasRestarted() {
+				d := pod.CreationTimestamp.Time.Sub(virtualmachine.CreationTimestamp.Time)
```

Comment:
I am slightly concerned about is that different `CreationTimestamp's can be recorded by different nodes in the cluster, thus this value can be inconsistent. I hope it is not noticeable on our scale.

Reply:
That's a valid concern. These values are created by k8s API server and we should have clock sync on them (infra team should know the exact details here).